### PR TITLE
Rewrite the tests in o.j.t.runtime.as.ui.bot.test to be parametized and

### PR DESCRIPTION
**Pull request by robstryker**
much smaller

### CODE REVIEW

**Comment by robstryker** [comment]
The code works here and it works in eclipse, but I'm having problems with command-line configuration :(  Also I was exhausted after pulling a bunch of 12 hour days.

**Comment by robstryker** [comment]
/bin/sh /usr/bin/mvn clean verify -P smoke -DskipTests=false -Dreq.java8.home=/home/rob/apps/java/java8/jdk1.8.0_05/ -Dreq.java7.home=/home/rob/apps/java/jdk1.7.0_67/ -Dreq.java6.home=/home/rob/apps/java/jdk1.6


This is the cmd line

**Comment by rawagner** [comment]
@mmalina please review

File: tests/org.jboss.tools.runtime.as.ui.bot.test/launchers/prepare_workspace/pom.xml [hunk 1]
```diff
@@ -7,36 +7,5 @@
 	<version>0.0.0-SNAPSHOT</version>
```
Comment (rawagner) [inline tests/org.jboss.tools.runtime.as.ui.bot.test/launchers/prepare_workspace/pom.xml]: Cant we remove this file completely ?

**Comment by mmalina** [comment]
Rob says this will need changes based on his conversation with Rasto. So I will wait :)

File: tests/org.jboss.tools.runtime.as.ui.bot.test/src/org/jboss/tools/runtime/as/ui/bot/test/parametized/server/ServerRuntimesTest.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+ */
+
+
+@RunWith(RedDeerSuite.class)
+@UseParametersRunnerFactory(ParameterizedRequirementsRunnerFactory.class)
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)//first acquireAndDetect, then detect, then operate
+@DisableSecureStorage
+public class ServerRuntimesTest {
+
+    @Parameters
```
Comment (rhopp) [inline tests/org.jboss.tools.runtime.as.ui.bot.test/src/org/jboss/tools/runtime/as/ui/bot/test/parametized/server/ServerRuntimesTest.java]: add `(name="{0}") so we can see in junit results for which runtime the test failed?

**Comment by rhopp** [comment]
I haven't had chance to go through in detail yet. But for now I think we should leave RuntimeDuplications test (or at least create new task somewhere (jira?) to re-create it). I also added a comment to ServerRuntimesTest to add name to parameters.

**Pull request closed without merging**